docs/source/release_notes/index.rst

```diff
@@ -37,3 +44,16 @@ The following pages summarize what is new in Avocado:
    0_27_0
    0_26_0
    0_25_0
+
+Long Term Stability Releases
+============================
+
+The LTS releases are regular releases considering the release cycle, but
+they are flagged as supported for a period of about 18 months after
```

Comment:
the `they are flagged as supported` sounds a bit strange to me. How about `but new branch is created and bugfixes are backported on demand for 18 months after the release` (IIRC we mentioned it's 18 months, not about 18 months)